src/jsonapi.js

```diff
@@ -200,11 +202,11 @@ export const deleteEntity = (entity, {
   return (dispatch, getState) => {
     dispatch(apiWillDelete(entity));
 
-    const { host: apiHost, path: apiPath, accessToken } = getState().api.endpoint;
-    const endpoint = `${apiHost}${apiPath}/${entity.type}/${entity.id}`;
+    const { axiosConfig } = getState().api.endpoint;
```

Comment:
`axiosConfig` not passed to `apiRequest`, thus it's missing baseURL and other global config, like Authorization header.